tools/db_crashtest.py

```diff
@@ -51,6 +51,7 @@
     # Disabled compression_parallel_threads as the feature is not stable
     # lambda: random.choice([1] * 9 + [4])
     "compression_parallel_threads": 1,
+    "compression_max_dict_buffer_bytes": lambda: 4096 * random.randint(0, 32),
```

Comment:
There should be no reason to use only "friendly" sizes in the stress test, right?

Reply:
Right. Not sure of any particularly unfriendly value but we can make the range wider. I changed it to one less than powers of two from 1 -> 40.